### PR TITLE
feat(rgbpp-sdk): Add rgbpp sub package

### PR DESCRIPTION
**Pull request by duanyytop**
## Main Changes

- Add rgbpp sub package to RGB++ SDK
- Export common functions and types from rgbpp root path
- Export functions and types of sub packages from sub paths

### CODE REVIEW

File: packages/rgbpp/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export { remove0x } from '@rgbpp-sdk/ckb';
+export { ErrorCodes, ErrorMessages } from '@rgbpp-sdk/btc';
+export { ErrorCodes as ServiceErrorCodes, ErrorMessages as ServiceErrorMessage } from '@rgbpp-sdk/service';
+
+export * from '@rgbpp-sdk/ckb';
```
Comment (ShookLyngs) [inline packages/rgbpp/src/index.ts:5]: Should each sub package be exported in its own path? For example:
```ts
import { sendBtc } from 'rgbpp/btc';
import { genBtcJumpCkbVirtualTx } from 'rgbpp/ckb';
```

File: packages/rgbpp/src/index.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+export { remove0x } from '@rgbpp-sdk/ckb';
+export { ErrorCodes, ErrorMessages } from '@rgbpp-sdk/btc';
```
Comment (ShookLyngs) [inline packages/rgbpp/src/index.ts:2]: Maybe errors in the BTC lib should also be renamed 
```suggestion
export { ErrorCodes as BtcErrorCodes, ErrorMessages as BtcErrorMessages } from '@rgbpp-sdk/btc';
```

**Review by ShookLyngs — CHANGES_REQUESTED** [review]

File: packages/rgbpp/src/index.ts (hunk 2 above)
Comment (ShookLyngs) [reply packages/rgbpp/src/index.ts:2]: Okay, maybe if each sub-package is exported from its own path, the issue can also be resolved.

File: README.md [hunk 3]
```diff
@@ -4,6 +4,7 @@ This repository offers utilities for Bitcoin and RGB++ asset integration.
 
 ### Packages in this repository
 
+- [rgbpp-sdk](./packages/rgbpp): A root package to integrate of common functions from the RGB++ SDK sub-packages(btc/ckb/service).
```
Comment (ShookLyngs) [inline README.md:7]:
```suggestion
- [rgbpp](./packages/rgbpp): A root package to integrate of common functions from the RGB++ SDK sub-packages(btc/ckb/service).
```

**Review by ShookLyngs — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -4,6 +4,7 @@ This repository offers utilities for Bitcoin and RGB++ asset integration.
 
 ### Packages in this repository
 
+- [rgbpp](./packages/rgbpp): A root package to integrate of common functions from the RGB++ SDK sub-packages(btc/ckb/service).
```
Comment (Flouse) [inline README.md:7]: ### A Thought
What about revealing this root package in `/README` after some testing, after the code examples is updated?

**Review by Flouse — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (duanyytop) [reply README.md:7]: Make sense

File: packages/rgbpp/src/ckb.ts [hunk 5]
```diff
@@ -0,0 +1 @@
+export * from '@rgbpp-sdk/service';
```
Comment (ahonn) [inline packages/rgbpp/src/ckb.ts:1]: The file name seems to be inconsistent with the exported package

File: packages/rgbpp/src/service.ts [hunk 6]
```diff
@@ -0,0 +1 @@
+export * from '@rgbpp-sdk/btc';
```
Comment (ahonn) [inline packages/rgbpp/src/service.ts:1]: Same as above, this should be `@rgbpp-sdk/service`?

File: packages/rgbpp/src/ckb.ts (hunk 5 above)
Comment (duanyytop) [reply packages/rgbpp/src/ckb.ts]: Fixed

File: packages/rgbpp/src/service.ts (hunk 6 above)
Comment (duanyytop) [reply packages/rgbpp/src/service.ts]: Fixed

**Review by ahonn — APPROVED** [review]

**Review by yuche — APPROVED** [review]

**Review by ShookLyngs — APPROVED** [review]

**Pull request merged**